src/main/webapp/utils.ts

```diff
@@ -14,9 +14,13 @@
 
 import { ProjectGraphData } from './model/project-graph-data';
```

Comment:
Certainly! I plan on switching over to using Angular services (which I'm doing in my other branch) which should make this much cleaner. I think it would save time to leave `utils` as-is for now, and shift that refactoring step to my next branch. Does that sound reasonable?

Reply:
Since I already have work done in the other branch (including some services) it would save me quite a bit of time. That branch is also almost ready for a PR anyway, so `main` would only be behind for a day or so